app/src/main/java/com/zulip/android/activities/DevAuthActivity.java

```diff
@@ -0,0 +1,147 @@
+package com.zulip.android.activities;
+
+import android.app.Activity;
+import android.app.ProgressDialog;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v4.content.ContextCompat;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.TextView;
+
+import com.zulip.android.R;
+import com.zulip.android.networking.AsyncLogin;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DevAuthActivity extends Activity {
+    RecyclerView recyclerView;
+    private ProgressDialog connectionProgressDialog;
+
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_dev_auth);
+        recyclerView = (RecyclerView) findViewById(R.id.devAuthRecyclerView);
+        String json = getIntent().getStringExtra("emails_json");
```

Comment:
might be nice to move `emails_json` to a constant somewhere that can be referenced by all uses across the app.